.github/workflows/ci.yaml

```diff
@@ -21,14 +21,19 @@ env:
 jobs:
   ci:
     uses: uptick/actions/.github/workflows/ci.yaml@main
-    secrets: inherit
+    secrets:
+      SECRET_ENV: "${{ secrets.DEPLOY_KEY }}"
     with:
       python: true
       python-version: 3.11
       praise-on-fix: false
       poetry: true
       poetry-install-command: "poetry install"
       command: |
+        echo "$SECRET_ENV" > /tmp/deploy_key
+        chmod 600 /tmp/deploy_key
+        GIT_SSH_COMMAND="ssh -i /tmp/deploy_key -o StrictHostKeyChecking=accept-new" git clone git@github.com:uptick/splat-private.git
```

Comment:
I think we don't need it for this job no?

Reply:
We do, because the command also builds the docker image locally and it fails without `splat-private` content

Reply:
Are we building the docker image? I just see mose run ci but no docker

Reply:
Mmm i guess we are. Ok